app/src/main/java/com/vci/vectorcamapp/intake/presentation/IntakeScreen.kt

```diff
@@ -396,8 +396,20 @@ fun IntakeScreen(
                 state.surveillanceForm?.let { surveillanceForm ->
                     TextEntryField(
                         label = "Number of People Living in the House",
-                        value = if (surveillanceForm.numPeopleSleptInHouse == 0) "" else surveillanceForm.numPeopleSleptInHouse.toString(),
-                        onValueChange = { onAction(IntakeAction.EnterNumPeopleSleptInHouse(it.filter { character -> character.isDigit() })) },
+                        value = surveillanceForm.numPeopleSleptInHouse.toString(),
+                        onValueChange = { newValue ->
+                            val oldValue = surveillanceForm.numPeopleSleptInHouse.toString()
+                            val filteredNewValue = newValue.filter { it.isDigit() }
+
+                            val finalValueString = if (oldValue == "0" && filteredNewValue.length > 1) {
+                                filteredNewValue.filter { it != '0' }
+                            } else {
+                                filteredNewValue
+                            }
+
+                            val normalizedFinalValue = finalValueString.toIntOrNull()?.toString() ?: "0"
```

Comment:
Question: Should we keep the value zero by default or the empty string? I am in favor of the latter option because otherwise the user might mistakenly submit zero for the numbers.